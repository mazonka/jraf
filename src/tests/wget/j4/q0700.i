command=jw md 0 /.jraf.sys/users : login admin@server.com * : read f82d3bb17612458c /.jraf.sys/login : logout f82d3bb17612458c : login admin@server.com * : read 5aeb1f6c5cd9c44a /.jraf.sys/login : logout 5aeb1f6c5cd9c44a