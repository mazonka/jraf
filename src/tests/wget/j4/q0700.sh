#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
ls -1 $wr/.jraf.sys/login/
rm -rf $wr



