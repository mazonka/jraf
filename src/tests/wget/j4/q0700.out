OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK 0 -1 1 0 16 f82d3bb17612458c : OK : OK http://127.0.0.1:16000/jraf? : OK 0 -1 1 0 16 5aeb1f6c5cd9c44a : OK
==
